Assets/Scripts/Player/Development/ResourceEditor.cs

```diff
@@ -0,0 +1,137 @@
+using UnityEngine;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+[CustomEditor(typeof(ResourceSpawner)), CanEditMultipleObjects]
+public class ResourceEditor : Editor {
+
+    private ResourceSpawner _resourceSpawner;
+
+    private bool _spawningStarted = false;
+
+    private void OnEnable() {
+        _resourceSpawner = target as ResourceSpawner;
+
+    }
+
+#if UNITY_EDITOR
+    public void OnSceneGUI() {
+        if (_resourceSpawner.transform == Selection.activeTransform) {
+            DrawSpawnHandles();
+        }
+
+        EditorGUI.BeginChangeCheck();
+        var typeToSpawn = _resourceSpawner.TypeToSpawn;
+        if (EditorGUI.EndChangeCheck()) {            
+            Undo.RecordObject(_resourceSpawner, "Spawner");
+            var viewedModelFilter = ((MeshFilter)_resourceSpawner.GetComponent("MeshFilter"));
```

Comment:
Outer braces unnecessary

Reply:
Also, bigbrick seems to not have been committed

Reply:
the whole thing was just a PoC to switch out meshes, which I scrapped later, but forgot to delete that code